src/ring_theory/power_series/well_known.lean

```diff
@@ -119,4 +119,15 @@ end
 theorem exp_mul_exp_neg_eq_one [algebra ℚ A] : exp A * eval_neg_hom (exp A) = 1 :=
 by convert exp_mul_exp_eq_exp_add (1 : A) (-1); simp
 
+/-- Shows that $(e^{X})^k = e^{kX}$. -/
+theorem exp_pow_eq_rescale_exp  [algebra ℚ A] (k : ℕ) : (exp A)^k = rescale (k : A) (exp A) :=
+begin
+  induction k with k h,
+  { simp only [rescale_zero, constant_coeff_exp, function.comp_app, map_one, cast_zero,
+  pow_zero, coe_comp], },
+  rw [succ_eq_add_one, cast_add, ←exp_mul_exp_eq_exp_add (k : A), ←h],
+  simp only [cast_one, id_apply, rescale_one],
```

Comment:
Nice simplification! Here's some more golfing:
```suggestion
      pow_zero, coe_comp], },
  simpa only [succ_eq_add_one, cast_add, ←exp_mul_exp_eq_exp_add (k : A), ←h, cast_one,
    id_apply, rescale_one] using pow_succ' (exp A) k,
```

Reply:
In fact, the whole proof can even be:
```lean
begin
  induction k with k h, { simp, },
  simpa [←exp_mul_exp_eq_exp_add (k : A), ←h] using pow_succ' (exp A) k,
end
```
But this takes a noticeable amount of time (~2.48s) to compile in my editor (using `set_option profiler true`), whereas the `simp only` version takes 456 ms.